subsys/modem/backends/modem_backend_uart_isr.c

```diff
@@ -189,6 +240,23 @@ struct modem_pipe_api modem_backend_uart_isr_api = {
 	.close = modem_backend_uart_isr_close,
 };
 
+#if CONFIG_MODEM_STATS
+static void init_stats(struct modem_backend_uart *backend)
+{
+	char name[CONFIG_MODEM_STATS_BUFFER_NAME_SIZE];
+	uint32_t receive_buf_size;
+	uint32_t transmit_buf_size;
+
+	receive_buf_size = get_receive_buf_capacity(backend);
```

Comment:
`receive_buf_size = get_receive_buf_capacity(backend)` while there is a similar function that is called `get_receive_buf_size()`. This is correct here, but confusing!
(The use of `size` coming from the `ring_buffer` API doesn't help the confusion around `size`, but it could help to stick to one naming.)

Reply:
Agree 100%, I was wondering if I should update the modules to use capacity/size everywhere, as C++ standard lib uses size/lenght interchangeably to mean length, and capacity for size, but the size of a buffer is commonly understood as the capacity, and length for used space.

I have updated everything to use size and length, `capacity` is no longer used